src/progressbar/progressbar.component.html

```diff
@@ -1,6 +1,6 @@
-<bar [type]="type" [value]="_value" *ngIf="!isStacked">
+<bar [type]="type" [value]="_value" [max]="_max" *ngIf="!isStacked">
   <ng-content></ng-content>
 </bar>
 <ng-template [ngIf]="isStacked">
-  <bar *ngFor="let item of _value" [type]="item.type" [value]="item.value">{{ item.label }}</bar>
+  <bar *ngFor="let item of _value" [type]="item.type" [value]="item.value" [max]="item.max">{{ item.label }}</bar>
```

Comment:
Hello!
I'm afraid this fix has broken the `max` attribute when using a "stacked" progress bar. This small piece of code requires each item of `BarValue` to have the same `max` attribute, rather than using the `max` input of the `ProgressbarComponent`.